src/pybind/mgr/dashboard/frontend/src/styles/ceph-custom/_forms.scss

```diff
@@ -18,7 +18,7 @@
 
   &:focus {
     border-color: rgba(vv.$primary, 0.8);
-    box-shadow: inset 0 1px 1px rgba(vv.$black, 0.75), 0 0 8px 2px rgba(vv.$primary, 0.5);
+    box-shadow: 0 0 3px 2px rgba(vv.$primary, 0.5);
```

Comment:
After this change the `box-shadow: 0 0 3px 2px rgba(vv.$primary, 0.5);` line can be removed completely, because it's very similar to the bootstraps default.

Reply:
Update: Could you please provide some screenshots of this change that we can see the difference?

Reply:
@Exotelis That was actually done here. https://github.com/ceph/ceph/pull/37769